great-idea-website/css/index.css

```diff
@@ -63,4 +63,92 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+.container {
+	width: 878px;
+	margin: auto;
+}
+/* header */
+	header nav {
+		display: inline-block;
+		margin-top: 35px;
+		margin-left: 40px;
+	}
+
+	header nav a {
+		margin: 0 20px;
+		color: gray;
+		text-decoration: none;
+	}
+
+	header img.logo {
+		margin-left: 50px;
+	}
+/* /header */
+
+/* cta */
+	.cta-text {
+		display: inline-block;
+		font-size: 70px;
+		text-align: center;
+		vertical-align: top;
+		padding: 55px;
+		margin-left: 25px;
+		margin-top: 30px;
+	}
+	.cta-text button {
+		font-size: 20px;
+		background-color: white;
```

Comment:
The button in the design file is pretty flat. To achieve the same effect, you can try declaring a new value for the `border` property to make the border just a square instead of this 3D shadow effect thing. You can also try removing `box-shadow`.